lib/inc/internal/facts/linux/virtualization_resolver.hpp

```diff
@@ -7,20 +7,36 @@
 #include "../resolvers/virtualization_resolver.hpp"
 #include <string>
 
+using namespace std;
+
 namespace facter { namespace facts { namespace linux {
 
     /**
      * Responsible for resolving virtualization facts.
      */
     struct virtualization_resolver : resolvers::virtualization_resolver
     {
-     protected:
+     public:
```

Comment:
Protected functions can be overridden when you subclass. You may need to modify how you inherit by specifying something like `struct my_virt_resolver : public virtualization_resolver`.